lib/linguist/languages.yml

```diff
@@ -2423,6 +2423,16 @@ Jolie:
   ace_mode: text
   tm_scope: source.jolie
   language_id: 998078858
+Jsonnet:
+  color: "#0064bd"
+  type: programming
+  ace_mode: text
+  extensions:
+  - ".jsonnet"
+  - ".libjsonnet"
```

Comment:
🔥 this extension for now - it's no where near prevalent enough at only 10 files.

Reply:
Done.